### PR TITLE
Download Manual debug

### PR DESCRIPTION
**Pull request by daschuer**
This reveals: 
```
ssl peer certificate or ssh remote key was not ok
```

I have no idea how to fix this. Any ideas?

### CODE REVIEW

**Comment by Swiftb0y** [comment]
Maybe the time on the windows VM is wrong? Did the cert algorithm change recently? Did the VM change recently? when exactly did the windows build start failing?

**Comment by daschuer** [comment]
It turns out that this is a cmake 3.26.4 issue. A downgrade to the minimum rewired 3.21 for windows works.

**Comment by daschuer** [comment]
OK this is ready. I have pinned the Windows CMake version to the minimum required 3.21 and it works.

**Comment by Swiftb0y** [comment]
Is there an upstream issue that tracks the problem for the affected CMake version?

File: .github/workflows/build.yml [hunk 1]
```diff
@@ -181,14 +181,23 @@ jobs:
...
         if: runner.os == 'macOS'
         with:
           # This should always match the minimum required version in
           # our CMakeLists.txt
           cmake-version: "3.19.x"
 
+      - name: "[Windows] Set up cmake"
+        uses: jwlawson/actions-setup-cmake@v1.13
+        # Ubuntu 22.04 should use the CMake version from the repos.
+        if: runner.os == 'macOS'
```
Comment (Swiftb0y) [inline .github/workflows/build.yml:194]: Copy-paste error? 
```suggestion
        if: runner.os == 'Windows'
```

**Comment by JoergAtGithub** [comment]
Are you aware that the certificate of mixxx.org changed? This is what my Firefox reports: 
grafik

Do we maybe cache the old one?

**Comment by Swiftb0y** [comment]
> Are you aware that the certificate of mixxx.org changed?

Yes, thats why I asked for the time on the windows VM... But I doubt that's it.

**Comment by JoergAtGithub** [comment]
On my local Windows11 system the download succeeds:
```
1> [CMake] -- Downloading manual from https://downloads.mixxx.org/manual/2.4/mixxx-manual-2.4-en.pdf...
1> [CMake] -- [download 0% complete]
1> [CMake] -- [download 1% complete]
1> [CMake] -- [download 2% complete]
1> [CMake] -- [download 3% complete]
1> [CMake] -- [download 4% complete]
1> [CMake] -- [download 5% complete]
1> [CMake] -- [download 6% complete]
1> [CMake] -- [download 7% complete]
1> [CMake] -- [download 8% complete]
1> [CMake] -- [download 9% complete]
1> [CMake] -- [download 10% complete]
1> [CMake] -- [download 11% complete]
1> [CMake] -- [download 12% complete]
1> [CMake] -- [download 13% complete]
1> [CMake] -- [download 14% complete]
1> [CMake] -- [download 15% complete]
1> [CMake] -- [download 16% complete]
1> [CMake] -- [download 17% complete]
1> [CMake] -- [download 18% complete]
1> [CMake] -- [download 19% complete]
1> [CMake] -- [download 20% complete]
1> [CMake] -- [download 21% complete]
1> [CMake] -- [download 22% complete]
1> [CMake] -- [download 23% complete]
1> [CMake] -- [download 24% complete]
1> [CMake] -- [download 25% complete]
1> [CMake] -- [download 26% complete]
1> [CMake] -- [download 27% complete]
1> [CMake] -- [download 28% complete]
1> [CMake] -- [download 29% complete]
1> [CMake] -- [download 30% complete]
1> [CMake] -- [download 31% complete]
1> [CMake] -- [download 32% complete]
1> [CMake] -- [download 33% complete]
1> [CMake] -- [download 34% complete]
1> [CMake] -- [download 36% complete]
1> [CMake] -- [download 38% complete]
1> [CMake] -- [download 41% complete]
1> [CMake] -- [download 42% complete]
1> [CMake] -- [download 43% complete]
1> [CMake] -- [download 44% complete]
1> [CMake] -- [download 45% complete]
1> [CMake] -- [download 46% complete]
1> [CMake] -- [download 47% complete]
1> [CMake] -- [download 48% complete]
1> [CMake] -- [download 49% complete]
1> [CMake] -- [download 50% complete]
1> [CMake] -- [download 51% complete]
1> [CMake] -- [download 52% complete]
1> [CMake] -- [download 53% complete]
1> [CMake] -- [download 54% complete]
1> [CMake] -- [download 55% complete]
1> [CMake] -- [download 56% complete]
1> [CMake] -- [download 57% complete]
1> [CMake] -- [download 58% complete]
1> [CMake] -- [download 59% complete]
1> [CMake] -- [download 60% complete]
1> [CMake] -- [download 61% complete]
1> [CMake] -- [download 62% complete]
1> [CMake] -- [download 63% complete]
1> [CMake] -- [download 64% complete]
1> [CMake] -- [download 65% complete]
1> [CMake] -- [download 66% complete]
1> [CMake] -- [download 67% complete]
1> [CMake] -- [download 68% complete]
1> [CMake] -- [download 69% complete]
1> [CMake] -- [download 70% complete]
1> [CMake] -- [download 71% complete]
1> [CMake] -- [download 72% complete]
1> [CMake] -- [download 73% complete]
1> [CMake] -- [download 74% complete]
1> [CMake] -- [download 75% complete]
1> [CMake] -- [download 76% complete]
1> [CMake] -- [download 77% complete]
1> [CMake] -- [download 78% complete]
1> [CMake] -- [download 79% complete]
1> [CMake] -- [download 80% complete]
1> [CMake] -- [download 81% complete]
1> [CMake] -- [download 82% complete]
1> [CMake] -- [download 83% complete]
1> [CMake] -- [download 84% complete]
1> [CMake] -- [download 85% complete]
1> [CMake] -- [download 86% complete]
1> [CMake] -- [download 87% complete]
1> [CMake] -- [download 89% complete]
1> [CMake] -- [download 90% complete]
1> [CMake] -- [download 91% complete]
1> [CMake] -- [download 92% complete]
1> [CMake] -- [download 93% complete]
1> [CMake] -- [download 94% complete]
1> [CMake] -- [download 95% complete]
1> [CMake] -- [download 97% complete]
1> [CMake] -- [download 98% complete]
1> [CMake] -- [download 99% complete]
1> [CMake] -- [download 100% complete]
```

**Comment by JoergAtGithub** [comment]
LGTM! Thank you!

**Comment by Swiftb0y** [comment]
> Is there an upstream issue that tracks the problem for the affected CMake version?

^^?

**Comment by daschuer** [comment]
This is the upstream bug:
https://gitlab.kitware.com/cmake/cmake/-/issues/24147

**Comment by Swiftb0y** [comment]
Thank you. so we should be pin cmake 3.26. instead as well once available.

**Comment by daschuer** [comment]
I think it is better to pin the minimum working version like ke done here.
This verifies that nothing from a newer version is used.

**Comment by Swiftb0y** [comment]
But then we won't know if something else in a new version broke. If we notice it too late and nobody has time to fix it, we'll be stuck on an old version. Shouldn't the `cmake_minimum_required` in the CMakeLists be sufficient that we're not using features that require a newer version?

**Pull request merged**